_includes/api/en/5x/req-secure.md

```diff
@@ -1,3 +1,8 @@
 <h3 id='req.secure'>req.secure</h3>
  
- A Boolean property that is true if a TLS connection is established. Equivalent to `req.protocol === 'https'`.
+ A Boolean property that is true if a TLS connection is established. Equivalent to the following:
+
+```js
+<!-- eslint-disable no-unused-expressions -->
```

Comment:
Apologies, I should have been more specific. The eslint comment needs to be just above the code block, the linter is choking on finding HTML in the js code block.

Thank you for the quick change, btw!